packages/suite/src/components/firmware/FirmwareProgressBar.tsx

```diff
@@ -68,20 +29,29 @@ export const FirmwareProgressBar = () => {
     const isDone = progress === 100;
 
     return (
-        <Wrapper>
-            {operation && (
-                <Label>
-                    <Translation id={mapOperationToTransaltionId[operation]} />
-                </Label>
-            )}
-            <StyledProgressBar value={progress} />
-            <Percentage>
-                {isDone ? (
-                    <Icon name="check" color={theme.legacy.TYPE_GREEN} size={24} />
-                ) : (
-                    `${progress} %`
-                )}
-            </Percentage>
-        </Wrapper>
+        <Box width="100%">
```

Comment:
I'm not happy about this one, but I would need to update alignItems:stretch in `OnboardingStepBox` which could lead many issues across the app. Not brave enough right now :D let's leave it for broader refactoring.